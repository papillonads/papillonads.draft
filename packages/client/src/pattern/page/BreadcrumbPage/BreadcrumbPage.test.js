import React from 'react'
import breadcrumbPageDefault from './BreadcrumbPage.render'

jest.mock('react-apollo', () => {
  return {
    Query: () => <div />,
  }
})

describe('<BreadcrumbPage />', () => {
  describe('Snaphot', () => {
    test('must match breadcrumbPageDefault', () => {
      expect(global.renderToJSON(breadcrumbPageDefault)).toMatchSnapshot()
    })
  })
})
