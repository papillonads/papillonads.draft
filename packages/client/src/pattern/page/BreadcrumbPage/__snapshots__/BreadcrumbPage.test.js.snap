// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BreadcrumbPage /> Snaphot must match breadcrumbPageDefault 1`] = `<div />`;
